_docs/query-data/query-a-file-system/040-querying-directories.md

```diff
@@ -152,3 +152,22 @@ Starting in Drill 1.16, Drill uses a Value operator instead of a Scan operator t
  
 You can use [query directory functions]({{site.baseurl}}/docs/query-directory-functions/) to restrict a query to one of a number of subdirectories and to prevent Drill from scanning all data in directories.
 
+##Querying Multiple Directories in One Query by Wildcard Matching
```

Comment:
Put a whitespace between the `##` and `Querying`.

Reply:
Hi cong, I did the change as you required. Please take a look